lighthouse-plugin-ad-speed-insights/messages/en-US.js

```diff
@@ -0,0 +1,163 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const Messages = {
+  AUDITS: {
+    'ad-blocking-tasks': {
+      title: 'No long tasks seem to block ad-related network requests',
+      failureTitle: 'Long tasks are blocking ad-related network requests',
+      description: 'Tasks blocking the main thread can delay the ad related ' +
```

Comment:
I would favor eliminating line breaks in this file.

On second thought this would be better as JSON so that it's pure configuration. WDYT?